QuasarHelios2015/src/org/team1540/quasarhelios/Rollers.java

```diff
@@ -0,0 +1,60 @@
+package org.team1540.quasarhelios;
+
+import ccre.channel.BooleanInput;
+import ccre.channel.BooleanOutput;
+import ccre.channel.BooleanStatus;
+import ccre.channel.EventOutput;
+import ccre.channel.FloatOutput;
+import ccre.channel.FloatStatus;
+import ccre.ctrl.FloatMixing;
+import ccre.igneous.Igneous;
+
+public class Rollers {
+	public final static BooleanStatus direction = new BooleanStatus(true);
+	public final static FloatStatus speed = new FloatStatus(0.0f);
+	
+	public static FloatOutput armRollers = FloatMixing.combine(Igneous.makeTalonMotor(5, Igneous.MOTOR_REVERSE, 0.1f), Igneous.makeTalonMotor(6, Igneous.MOTOR_FORWARD, 0.1f)); 
+	public static FloatOutput frontRollers = Igneous.makeTalonMotor(7, Igneous.MOTOR_FORWARD, 0.1f);
+	public static FloatOutput internalRollers = FloatMixing.combine(Igneous.makeTalonMotor(8, Igneous.MOTOR_REVERSE, 0.1f), Igneous.makeTalonMotor(9, Igneous.MOTOR_FORWARD, 0.1f)); 
+	
+	public static FloatOutput externalRollers = FloatMixing.combine(armRollers, frontRollers);
+	public static FloatOutput allRollers = FloatMixing.combine(externalRollers, internalRollers);
+	
+	public static BooleanInput toggleRollersButton;
+	public static BooleanInput runRollersButton;
+	
+	public static BooleanOutput runRollers = new BooleanOutput() {
+		public void set(boolean value) {
+			if (value) {
```

Comment:
Note that testing the value like this can mean that a single press of the button could theoretically result in this code getting pressed more than once. There is **no guarantee** that `set` will only be called with a different value than last time! `onPress`, `onRelease`, and `whenBooleanChanges` are useful Mixing methods for this - though that should be a moot point if you take an EventInput instead of a BooleanInput.